tests/two_sum_test.py

```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
+def test_returns_0_1_for_2_7_11_15_with_target_9():
+    # Arrange
+    n = [2, 7, 11, 15]
+    t = 9
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```

Comment:
Great code! Very clear cut tests. I wonder what other nominal cases we can think of.